api/Element.json

```diff
@@ -336,6 +336,57 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "delegatesFocus": {
+          "__compat": {
+            "description": "<code>delegatesFocus</code> option of <code>ShadowRootInit</code> dictionary.",
+            "support": {
+              "chrome": {
+                "version_added": true
```

Comment:
Chrome implements this [on `ShadowRoot`](https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/dom/shadow_root.idl).

Reply:
Ah yes, I'd forgotten about that property. I've added a page for it: https://developer.mozilla.org/en-US/docs/Web/API/ShadowRoot/delegatesFocus

The BCD for that was already written, which is cool. The delegatesFocus I am talking about in this PR is the delegateFocus option on Element.attachShadow's dictionary.

Reply:
I can't seem to be clear to save my life lately. Sorry about that. None of the three Chrome flavors  support this on Element.

Reply:
It is not a property hanging off element. It is a member of the ShadowRootInit dictionary, which attachShadow can take as a parameter. See https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/dom/shadow_root_init.idl?g=0

Reply:
I'm not a Shadow DOM expert. Are you telling me that Element can get this property with a call to attachShadow()?

Reply:
Read https://developer.mozilla.org/en-US/docs/Web/API/Element/attachShadow. delegateFocus is an option you can set when you attach a Shadow DOM to an element using attachShadow().

Later on, if you then want to query whether your attached shadow delegates focus, you can do so with https://developer.mozilla.org/en-US/docs/Web/API/ShadowRoot/delegatesFocus